text/0027-foreman-plugin-forge.md

```diff
@@ -0,0 +1,141 @@
+- Feature Name: A Forge for Foreman plugins
```

Comment:
+1 - But let's avoid the word "forge". It's already taken :)

How about Foundry? Fits the construction theme :)